.github/workflows/ccpp.yml

```diff
@@ -1,17 +1,175 @@
 name: C/C++ CI
 
 on: [push, pull_request]
+env:
+   TRAVIS: true
 
 jobs:
-  build:
+  unit_and_style:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: build
+      run:  scripts/build_ci.bash
+
+    - name: Check style
+      run:  make compliant && make shellcheck && make docs-coverage
+
+    - name: Check install
+      run:  sudo make install && make install-check
+
+    - name: Run unit tests
+      run:  make unit-check
+
+    - name: Print status
+      if: failure()
+      run: cat test-suite.log
+
+  test_update1:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: build
+      run:  scripts/build_ci.bash
+
+    - name: Run Check
+      run: |
+            UPDATE_TOTAL="$(find test/functional/update -name *.bats | wc -l)"
+            UPDATE_SUBGROUP1_TOTAL=17
+            UPDATE_SUBGROUP2_TOTAL="$((UPDATE_TOTAL - UPDATE_SUBGROUP1_TOTAL))"
+            UPDATE_SUBGROUP="$(find test/functional/update -name *.bats | head -n $UPDATE_SUBGROUP1_TOTAL | tr '\n' ' ')"
+            env TESTS="$UPDATE_SUBGROUP" make -e -j2 check
+
+    - name: Print status
+      if: failure()
+      run: cat test-suite.log
+
+  test_update2:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: build
+      run:  scripts/build_ci.bash
+
+    - name: Run Check
+      run: |
+            UPDATE_TOTAL="$(find test/functional/update -name *.bats | wc -l)"
+            UPDATE_SUBGROUP1_TOTAL=17
+            UPDATE_SUBGROUP2_TOTAL="$((UPDATE_TOTAL - UPDATE_SUBGROUP1_TOTAL))"
+            UPDATE_SUBGROUP="$(find test/functional/update -name *.bats | head -n $UPDATE_SUBGROUP2_TOTAL | tr '\n' ' ')"
```

Comment:
you are running the same tests in test_update1 and test_update2, you need to use tail in update2 instead of head:
`UPDATE_SUBGROUP="$(find test/functional/update -name *.bats | tail -n $UPDATE_SUBGROUP2_TOTAL | tr '\n' ' ')"`

Reply:
wow, that's true. failed on copy and paste